src/microbe_stage/components/StrainAffected.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Components
+{
+    using System;
+
+    /// <summary>
+    ///   Contains variables related to strain
+    /// </summary>
+    [JSONDynamicTypeAllowed]
+    public struct StrainAffected
+    {
+        /// <summary>
+        ///   True when sprinting or when strain is supposed to be otherwise generated
+        /// </summary>
+        public bool IsUnderStrain;
```

Comment:
Could you reorder these variables for memory efficiency? i.e. biggest variables first and this bool last as this takes just one byte so this being first causes 3 extra padding bytes to be added (in this case this doesn't matter but if there was a second bool in this struct then this would consume extra memory unnecessarily so it is good practice).